resources/repos.json

```diff
@@ -1005,6 +1005,22 @@
             }
         }
     },
+    "waleks/SMG:master": {
```

Comment:
Your username isn't waleks, it's waleks647.

The repository `waleks/SMG` does not exist:
https://github.com/waleks/SMG